alot/ui.py

```diff
@@ -157,6 +157,8 @@ def fire(_, cmdline):
                     return [cmdline[5:]]
 
             key = keys[0]
+            if len(key) > 1 and key[0].startswith('mouse'):
+                key = key[0]+' %i' % key[1]
```

Comment:
The formatting on this isn't quite right, we use spaces between operators.
so: `key = key[0] + ' %i' % key[1]`. Although I think that you can simplify this to `' '.join(key[:2])`